snippets/Timer.md

```diff
@@ -0,0 +1,49 @@
+### Timer
+
+Renders a timer component.
+
+- The timer state is initially set to zero 
+- When the `Tick!` button is clicked, `timer` is incremented periodically at the given `interval`
+- When the `Reset` button is clicked, the value of the timer is set to zero and the `setInterval` is cleared
+- The `setInterval` is cleared once the desired `time` is reached
+- `time` and `interval` are the required props
+
+```jsx
+class Timer extends Component {
+	constructor(props) {
+		super(props);
+		this.state = {timer: 0}
```

Comment:
Is there a more semantic name we can use for this? Don't get me wrong, timer is okay, but sometimes it's a little... confusing since it's current repetitions less than target repetitions

Reply:
Yes... This needs a better name.

Reply:
How about `Ticker` ?

Reply:
`ticker` seems a little better than timer

Reply:
I also agree that `Ticker` is slightly better.

Reply:
I committed that change